cluster/gce/gci/configure-helper.sh

```diff
@@ -1858,6 +1858,19 @@ function prepare-etcd-manifest {
   fi
   # Replace the volume host path.
   sed -i -e "s@/mnt/master-pd/var/etcd@/mnt/disks/master-pd/var/etcd@g" "${temp_file}"
+  # Replace the run as user and run as group
+  pod_run_as_user=""
+  pod_run_as_group=""
+  container_security_context=""
+  if [[ -n "${ETCD_RUNASUSER:-}" && -n "${ETCD_RUNASGROUP:-}" ]]; then
+    pod_run_as_user="\"runAsUser\": ${ETCD_RUNASUSER},"
+    pod_run_as_group="\"runAsGroup\": ${{ETCD_RUNASGROUP},"
+    container_security_context="\"securityContext\": {\"allowPrivilegeEscalation\": false, \"capabilities\": {\"drop\": [\"all\"]}},"
+    chown -R ${ETCD_RUNASUSER}:${ETCD_RUNASGROUP} /var/etcd/
```

Comment:
I would really like to avoid "chown" in the prepare manifests function.

Can we move this one to start-etcd-servers (also given that /var/etcd is actually shared between the two etcd's)?

Reply:
Fixed.